lighthouse-plugin-publisher-ads/messages/locales/en-XL.json

```diff
@@ -203,16 +203,16 @@
   "audits/duplicate-tags.js | title": {
     "message": "N̂ó d̂úp̂ĺîćât́ê t́âǵŝ f́ôún̂d́ îń âńŷ f́r̂ám̂é"
   },
-  "audits/first-ad-paint.js | description": {
+  "audits/first-ad-render.js | description": {
     "message": "T̂h́îś m̂ét̂ŕîć m̂éâśûŕêś t̂h́ê t́îḿê f́ôŕ t̂h́ê f́îŕŝt́ âd́ îf́r̂ám̂é t̂ó p̂áîńt̂ f́r̂óm̂ ṕâǵê ńâv́îǵât́îón̂. [Ĺêár̂ń m̂ór̂é](ĥt́t̂ṕŝ://d́êv́êĺôṕêŕŝ.ǵôóĝĺê.ćôḿ/p̂úb̂ĺîśĥér̂-ád̂ś-âúd̂ít̂ś/r̂éf̂ér̂én̂ćê/áûd́ît́ŝ/ḿêt́r̂íĉś)."
```

Comment:
Can you generate this with collect-strings.js?

Reply:
Done